src/editor/actions/change_index_action.cpp

```
@@ -14,7 +14,7 @@ ChangeIndexAction::~ChangeIndexAction() {
 
 void ChangeIndexAction::Do() {
     // update the node tree
-    auto parentNode = m_node->GetParent();
+    auto* parentNode = m_node->GetParent();
```

Clang (14.0.6)

[error] variable 'parentNode' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)